modules/dxp/apps/commerce/headless/headless-commerce-machine-learning-impl/src/main/java/com/liferay/headless/commerce/machine/learning/internal/resource/v1_0/AccountForecastResourceImpl.java

```diff
@@ -99,6 +114,33 @@
 			totalItems);
 	}
 
+	private void _createItem(AccountForecast item) throws Exception {
```

Comment:
@riccardo-ferrari can we rename `item` to `accountForecast`?

Reply:
Done, thank you